components/user_settings/notifications/desktop_notification_setting/desktop_notification_settings.tsx

```diff
@@ -353,7 +379,6 @@ export default class DesktopNotificationSettings extends React.PureComponent {
             <SettingItemMin
                 title={Utils.localizeMessage('user.settings.notifications.desktop.title', 'Desktop Notifications')}
                 describe={<FormattedMessage {...formattedMessageProps}/>}
-                focused={this.props.focused}
```

Comment:
did we forgot this prop?

Reply:
Nope, it was intentionally deleted it. Because, in `SettingItemMin` there is no focused prop. 🤓